misc/txt2pdf.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env python3
+
+import argparse
+import os
+import re
+from datetime import datetime
+
+from reportlab.lib import colors
+from reportlab.lib.pagesizes import LETTER
+from reportlab.lib.styles import ParagraphStyle, getSampleStyleSheet
+from reportlab.platypus import (
+    SimpleDocTemplate,
+    Paragraph,
+    Spacer,
+    Table,
+    TableStyle,
+    PageBreak,
+)
+from reportlab.lib.enums import TA_CENTER  # For center alignment of text
```

Comment:
Should keep all imports from the same module together. Here I have moved the `reportlab.lib.enums` import to underneath `reportlab.lib.styles`

```suggestion
from reportlab.lib import colors
from reportlab.lib.pagesizes import LETTER
from reportlab.lib.styles import ParagraphStyle, getSampleStyleSheet
from reportlab.lib.enums import TA_CENTER  # For center alignment of text
from reportlab.platypus import (
    SimpleDocTemplate,
    Paragraph,
    Spacer,
    Table,
    TableStyle,
    PageBreak,
)
```